packages/electron-network-status/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "@bugsnag/electron-network-status",
+  "version": "7.10.0",
+  "main": "network-status.js",
+  "description": "@bugsnag/electron package for watching network status",
+  "author": "Bugsnag",
+  "license": "MIT",
+  "homepage": "https://www.bugsnag.com/",
+  "devDependencies": {
+    "@bugsnag/core": "^7.10.0",
+    "@bugsnag/plugin-electron-client-state-manager": "^7.10.0"
+  },
```

The package manifest should define which files are included.
```suggestion
  },
  "files": [
    "*.js"
  ],
```

---

You can test this with `npm pack --dry-run <package>` to see which files would end up getting published to NPM